src/subject_level/getFFXdir.m

```diff
@@ -18,14 +18,18 @@
   % :returns: - :ffxDir: (string)
   %
 
+  glmDirName = createGlmDirName(opt, funcFWFM);
+
+  model = spm_jsonread(opt.model.file);
+  if ~isempty(model.Name) && ~strcmpi(model.Name, opt.taskName)
+    glmDirName = [glmDirName, '_desc-', converToValidCamelCase(model.Name)];
+  end
```

Comment:
if I understood right, if `model.Name` exists in `model.json` file, and it is different then than `taskname`, the pipeline will create a name e.g. `RhythmBlock-MNI-FWHM-0-` and append `model.Name` ?

Reply:
yup exactly, except that it will append it `_desc-[model.Name]`